lighthouse-core/scripts/dogfood-lhci.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+set -euox pipefail
```

Comment:
what's the benefit of doing this here instead of in the /bin/bash shebang? all my personal scripts look like this:

```
#!/bin/bash -ex

...
```

Reply:
no benefit I'm aware of, just personal habit I suppose.

is there a benefit to doing it in the shebang? :)

Reply:
i like one liners i guess

Reply:
oh after some googling I remembered I started this habit back at a company that used `#!/usr/bin/env bash` in all our scripts where the one-liner wasn't an option, so I guess that's one benefit of this `set` method is that it can be consistent across those two methods :)